android/expoview/src/main/java/versioned/host/exp/exponent/modules/api/screens/ScreenStackHeaderConfig.java

```diff
@@ -88,7 +88,7 @@ public ScreenStackHeaderConfig(Context context) {
     super(context);
     setVisibility(View.GONE);
 
-    mToolbar = ExpoViewBuildConfig.DEBUG ? new DebugMenuToolbar(context) : new Toolbar(context);
+    mToolbar = ExpoViewBuildConfig.getDEBUG() ? new DebugMenuToolbar(context) : new Toolbar(context);
```

Comment:
Would be nicer not to have a method named getDEBUG(). We might not control much of this code (can't move it to Kotlin) if it is vendored from RN Screens.

Reply:
I think this is the only place that it is used, and we should be able to convert this file in particular to kotlin. That being said I agree with the overall sentiment, so for other jvmstatic vars that are used widely I'd probably add a better-named method. That being said, I haven't yet hit any that can't be converted.